.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,31 @@
+on: [push, pull_request]
```

Comment:
I think this should allow you to approve tests to run on [pr's from forked repos](https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks)